velox/functions/sparksql/Arithmetic.h

```diff
@@ -219,6 +219,18 @@ struct HypotFunction {
   }
 };
 
+template <typename T>
+struct Log2Function {
+  FOLLY_ALWAYS_INLINE bool call(double& result, double a) {
+    double yAsymptote = 0.0;
```

Comment:
nit: this variable doesn't make the code more readable; consider dropping.

Reply:
just removed it.